typings/index-tests.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from "react";
```

Comment:
should add `@types/react` and `@types/react-dom` as dev dependencies to keep typescript compiler happy

Reply:
This library does not need to depend directly on `react-dom`, so it is not included.